src/js/components/ConnVersation.tsx

```diff
@@ -32,25 +28,23 @@ function filter(record: zng.Record, names: string[]) {
   return new zng.Record(cols, vals)
 }
 
-const ConnVersation = ({log, contextMenu}: Props) => {
```

Comment:
Here and in subsequent components, I've moved the contextMenu builder into a thunk to eliminate extra renders. And renamed the var "log" to "record".